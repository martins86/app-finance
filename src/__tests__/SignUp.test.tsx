import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import SignUp from '../pages/sign-up';

describe('Test <SignUp />', () => {
  test('Should match snapshot', () => {
    // Arrange
    const { container } = render(
      <Router>
        <SignUp />,
      </Router>
    );

    // Act & Assert
    expect(container).toMatchSnapshot();
  });
});
