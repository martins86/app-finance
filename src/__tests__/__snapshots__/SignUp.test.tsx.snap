// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <SignUp /> Should match snapshot 1`] = `
<div>
  <div
    class="sign-up"
  >
    <label
      class="label-system"
    >
      SISTEMA DE REGISTRO
    </label>
    <div
      class="sign-up-content"
    >
      <input
        class="input-primary"
        data-testid="email"
        placeholder="Digite seu E-mail"
        type="email"
        value=""
      />
      <input
        class="input-primary"
        data-testid="emailConf"
        placeholder="Confirme seu E-mail"
        type="email"
        value=""
      />
      <input
        class="input-primary"
        data-testid="password"
        placeholder="Digite sua Senha"
        type="password"
        value=""
      />
      <label
        class="label-error"
      />
      <button
        class="btn btn--primary btn--full-width"
        data-testid="button"
        title="Inscrever-se"
        type="button"
      >
        Inscrever-se
      </button>
      <label
        class="label-sign"
      >
        JÃ¡ tem uma conta?
        <strong
          class="text-strong-link"
        >
          <a
            href="/app-finance"
          >
            Entrar
          </a>
        </strong>
      </label>
    </div>
  </div>
</div>
`;
